agent/custom/auto_battle_repeat_action.py

```
@@ -0,0 +1,197 @@
+            )
+
+        ok = success_count == battle_count
+        mfaalog.info(
+            f"[auto_battle_repeat] end ok={ok} "
+            f"success_count={success_count}/{battle_count}"
+        )
+
+        # 通过 pipeline_override 输出结果信息到 GUI
+        if not ok:
+            display_text = (
+                f"战斗中断：已完成 {success_count}/{battle_count} 场"
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)